group_vars/docker-registry/storage_backend

```diff
@@ -1,17 +0,0 @@
----
-# default storage type is local (file).
-# storage_type: "local"
-
-# storage_type: "gcs"
-gcs_bucket: "gds-docker-registry"
-gcs_storage_path: "/"
-gcs_oauth2: false
-gcs_access_key: "{{ lookup('env', 'GCS_ACCESS_KEY') }}"
-gcs_secret_key: "{{ lookup('env', 'GCS_SECRET_KEY') }}"
```

Comment:
Maybe it's better to add google variables to site-gce.yml even if the object storage is not yet implemented on google, so not to lose track of them. Storage_type can stay "local" in there until the functionality is implemented, which will happen very soon.